doc/keycodes.txt

```diff
@@ -0,0 +1,162 @@
+Minetest Keycodes Reference
+===========================
+
+Key/Mouse codes as returned from formspec's `key_event` field.
+Also see <http://irrlicht.sourceforge.net/docu/_keycodes_8h_source.html>
+
+When choosing keys, try not to use arcane keys; instead, choose keys that are
+one every or most keyboards.
+
+* `KEY_LBUTTON`          = Left mouse button
```

Comment:
Minetest does not control this enum, so it would best to only refer to the Irrlicht docs and/or source.

Reply:
Irrlicht is unlikely to receive any more updates, so this is probably fine

Reply:
I think ease for modders is necessary.  And if Irrlicht receives an update, we'll have to patch it anyway because it would break any mods that rely on the current system.

Reply:
Actually, the names returned in `name` are returned in the `KeyPress` class which generates the names itself, so even if Irrlicht updates, the key names will stay the same.  So this isn't a problem.